src/Fmt_ast.ml

```diff
@@ -770,18 +770,19 @@ and fmt_package_type c ctx cnstrs =
   list_fl cnstrs fmt_cstr
 
 and fmt_row_field c ctx = function
-  | Rtag (name, atrs, const, typs) ->
+  | {prf_desc= Rtag (name, const, typs); prf_attributes= atrs; prf_loc} ->
       let c = update_config c atrs in
       let doc, atrs = doc_atrs atrs in
       hvbox 0
         ( hvbox 0
-            ( fmt_str_loc c ~pre:(str "`") name
-            $ fmt_if (not (const && List.is_empty typs)) " of@ "
-            $ fmt_if (const && not (List.is_empty typs)) " & "
-            $ list typs "@ & " (sub_typ ~ctx >> fmt_core_type c) )
+            ( Cmts.fmt c prf_loc
+            @@ ( fmt_str_loc c ~pre:(str "`") name
+               $ fmt_if (not (const && List.is_empty typs)) " of@ "
+               $ fmt_if (const && not (List.is_empty typs)) " & "
+               $ list typs "@ & " (sub_typ ~ctx >> fmt_core_type c) ) )
         $ fmt_attributes c ~key:"@" atrs
         $ fmt_docstring_padded c doc )
-  | Rinherit typ -> fmt_core_type c (sub_typ ~ctx typ)
+  | {prf_desc= Rinherit typ; _} -> fmt_core_type c (sub_typ ~ctx typ)
```

Comment:
We probably need to handle `prf_loc` and `prf_attributes` in the `Rinherit` case

Reply:
Fixed. Couldn't find the syntax either but I could fix a dropped comment.

Reply:
```
type t = [ `A | `B ]
type u = [ t [@test] | `C]
```

Reply:
Same as object types, the attributes is attached to the type `t`, not the `row_field`.